From: Boyuan Yang <byang@debian.org>
Date: Sun, 1 Sep 2019 13:57:17 -0400
Subject: pcf2vpnc: Use canonical shebang for perl

---
 pcf2vpnc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/pcf2vpnc b/pcf2vpnc
index 40da8a4..66db80d 100755
--- a/src/pcf2vpnc
+++ b/src/pcf2vpnc
@@ -1,4 +1,4 @@
-#!/usr/bin/env perl
+#!/usr/bin/perl
 # Stefan Tomanek <stefan@pico.ruhr.de>
 # updated by Wolfram Sang <ninja@the-dreams.de> on 21.10.06 and on 26.06.07
 ##
