Description: Remove obsolete TODO from vpnc(8) manpage
Author: Florian Schlichting <fschlich@zedat.fu-berlin.de>
Forwarded: no

--- a/vpnc.8.template
+++ b/vpnc.8.template
@@ -187,12 +187,6 @@
 disabling /etc/resolv.conf rewriting is documented in the README of the
 vpnc package.
 
-.SH TODO
-.PD 0
-Certificate support (Pre-Shared-Key + XAUTH is known to be insecure).
-.P
-Further points can be found in the TODO file.
-.PD
 
 .SH AUTHOR
 This man-page has been written by Eduard Bloch <blade(at)debian.org> and
