From: Mihai Maties <mihai@xcyb.org>
Date: Tue, 28 Dec 2010 11:09:26 +0100
Subject: Dead Peer Detection fix

Origin: http://lists.unix-ag.uni-kl.de/pipermail/vpnc-devel/2010-December/003492.html
---
 vpnc.c | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/vpnc.c b/vpnc.c
index 75d6562..7a46956 100644
--- a/vpnc.c
+++ b/vpnc.c
@@ -796,7 +796,9 @@ void dpd_ike(struct sa_block *s)
 		*/
 		s->ike.dpd_attempts = 6;
 		s->ike.dpd_sent = time(NULL);
+		s->ike.dpd_seqno = ntohl(s->ike.dpd_seqno);
 		s->ike.dpd_seqno++;
+		s->ike.dpd_seqno = htonl(s->ike.dpd_seqno);
 		send_dpd(s, 0, s->ike.dpd_seqno);
 	} else {
 		/* Our last dpd request has not yet been acked.  If it's been
@@ -1599,6 +1601,7 @@ static void do_phase1_am_packet2(struct sa_block *s, const char *shared_key)
 					if (s->ike.dpd_idle != 0) {
 						gcry_create_nonce(&s->ike.dpd_seqno, sizeof(s->ike.dpd_seqno));
 						s->ike.dpd_seqno &= 0x7FFFFFFF;
+						s->ike.dpd_seqno = htonl(s->ike.dpd_seqno);
 						s->ike.dpd_seqno_ack = s->ike.dpd_seqno;
 						s->ike.do_dpd = 1;
 						DEBUG(2, printf("peer is DPD capable (RFC3706)\n"));
