Description: fix spelling errors found by lintian
Author: Florian Schlichting <fschlich@zedat.fu-berlin.de>
Forwarded: yes

--- a/isakmp-pkt.c
+++ b/isakmp-pkt.c
@@ -816,7 +816,7 @@
 	}
 
 	DEBUG(3, printf("BEGIN_PARSE\n"));
-	DEBUG(3, printf("Recieved Packet Len: %zu\n", data_len));
+	DEBUG(3, printf("Received Packet Len: %zu\n", data_len));
 	fetchn(r->i_cookie, ISAKMP_COOKIE_LENGTH);
 	hex_dump("i_cookie", r->i_cookie, ISAKMP_COOKIE_LENGTH, NULL);
 	fetchn(r->r_cookie, ISAKMP_COOKIE_LENGTH);
--- a/vpnc.8.template
+++ b/vpnc.8.template
@@ -176,7 +176,7 @@
 make sure that files with colliding hashes have different names. The number
 can be derived from the certificate file itself:
 .P
-openssl x509 -subject_hash -noout -in /etc/vpnc/vpn-example-com.pem
+openssl x509 \-subject_hash \-noout \-in /etc/vpnc/vpn\-example\-com.pem
 
 See also the
 .B \-\-print\-config
@@ -241,7 +241,7 @@
 License can be found in /usr/share/common\-licenses/GPL.
 .SH "SEE ALSO"
 .BR pcf2vpnc (1),
-.BR cisco-decrypt (1),
+.BR cisco\-decrypt (1),
 .BR ip (8),
 .BR ifconfig (8),
 .BR route (1),
--- a/config.c
+++ b/config.c
@@ -319,7 +319,7 @@
 		"Script ",
 		"<command>",
 		"command is executed using system() to configure the interface,\n"
-		"routing and so on. Device name, IP, etc. are passed using enviroment\n"
+		"routing and so on. Device name, IP, etc. are passed using environment\n"
 		"variables, see README. This script is executed right after ISAKMP is\n"
 		"done, but before tunneling is enabled. It is called when vpnc\n"
 		"terminates, too\n",
